tribomobile/components/mainScreen.js

```diff
@@ -0,0 +1,142 @@
+import React from 'react';
+import {
+  StyleSheet,
+  View,
+  Text,
+  Image,
+  Dimensions,
+  TouchableOpacity,
+  TextInput,
+} from 'react-native';
+import MapView, {Marker} from 'react-native-maps';
+import menuImage from '../assets/menuImage.png';
+import food from '../assets/food.png';
+import store from '../assets/store.png';
+import user from '../assets/user1.png';
+import MapStyle from './MapStyle';
+import string from './constant';
+
+const MainScreen = () => {
+  function pressMenu() {
+    console.log('Se presiono el menu');
+  }
```

Comment:
is this method necessary?

Reply:
No, I'm testing for the method onpress={} in TouchableOpacity